docs/scarpet/Full.md

```diff
@@ -4756,6 +4756,9 @@ this should not happen, but let you be warned.
 Called right after a chunk at a given coordinate is loaded. All newly generated chunks are considered loaded as well.
  `x` and `z` correspond to the lowest x and z coordinates in the chunk.
 
+### `__on_chunk_unloaded(x, z)`
+Called right before a chunk at the given coordinates is unloaded. `x` and `z` correspond to the lowest x and z coordinates in the chunk.
+
```

Comment:
You don't need to modify Full.md, it gets done automatically

Reply:
Thank you for the nudge and sorry. I was thinking it could have been the case, but I didnt find anything quickly looking thru actions, my bad